test/unit/test_task.rb

```diff
@@ -38,6 +40,7 @@ def test_quality_task_some_suppressed
   def test_quality_task_some_not_installed
     get_test_object do |_task|
       setup_quality_task_mocks(uninstalled_tools: ['cane'])
+      @mocks[:which].expects(:which).with('cane').returns(nil)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.